modules/mono/mono_gd/gd_mono_method.h

```diff
@@ -38,15 +38,16 @@
 class GDMonoMethod : public IMonoClassMember {
 	StringName name;
 
-	int params_count;
+	uint16_t params_count;
```

Comment:
Same here.

Reply:
It's always initialized in the constructor, in the call to update the method signature. Is there a performance penalty in adding a default value here?